rebar.config

```diff
@@ -38,8 +38,10 @@
   undefined_function_calls,
   undefined_functions,
   locals_not_used,
-  deprecated_functions_calls,
-  deprecated_functions
+  deprecated_functions_calls
+  %% FIXME: This is turned of until we get rid of the use of sys:get_debug/3 in
```

Comment:
```suggestion
  %% FIXME: This is turned off until we get rid of the use of sys:get_debug/3 in
```

Reply:
Fixing some other issue with this PR now.